onnxruntime/python/tools/kernel_explorer/kernels/gemm_rocblas.cc

```diff
@@ -55,6 +57,10 @@ class RocBlasGemm : public GemmBase<T> {
     return {"Rocblas"};
   }
 
+  bool SelectImpl(const std::string& name) override {
+    return true;
```

Comment:
return name == "Rocblas"; 

This effectively doesn't really matter, but I think is more correct semantically.